src/Application.php

```diff
@@ -105,4 +100,32 @@ public function run()
 
         return $this;
```

Comment:
Can we change here to have:

`return $this->completeRequest($event);`

I saw it somewhere in another PR, but don't know where now.